test/ReactViews/Search/SearchBoxAndResultsSpec.tsx

```diff
@@ -106,4 +106,40 @@ describe("SearchBoxAndResults", function () {
     }).toThrow();
     expect(searchBox.props.value).toEqual(searchText);
   });
+
+  it("renders with an input & no SearchInDataCatalog when showSearchInCatalog is false", function () {
+    const searchText = "timmynook";
+    runInAction(() => {
+      viewState.searchState.locationSearchText = searchText;
+      viewState.searchState.showLocationSearchResults = true;
+      viewState.searchState.locationSearchResults = [];
+
+      Object.defineProperty(
+        viewState.terria.searchBarModel,
+        "showSearchInCatalog",
+        {
+          value: false,
+          writable: true
+        }
+      );
```

Comment:
@aclel - sorry one more tiny request. In Terria to correctly set a trait, you need to use this pattern instead of the `defineProperty()` call:

```js
      viewState.terria.searchBarModel.setTrait(
        CommonStrata.user,
        "showSearchInCatalog",
        false
      );
```

Reply:
I can fix this post merge :)

Reply:
Thanks @na9da! : ) I was wondering if there was another way to set that prop. I'll read the Trait docs.

Reply:
no problem, would you be able to make that change  and sync with the latest `main`? I'm not able to merge unless the branch is in sync with main. Thanks @aclel